modules/rhel-compute-updating.adoc

```diff
@@ -90,10 +88,10 @@ NAME                        STATUS                        ROLES    AGE    VERSIO
 mycluster-control-plane-0   Ready                         master   145m   v1.23.0
 mycluster-control-plane-1   Ready                         master   145m   v1.23.0
 mycluster-control-plane-2   Ready                         master   145m   v1.23.0
-mycluster-rhel7-0           NotReady,SchedulingDisabled   worker   98m    v1.23.0
-mycluster-rhel7-1           Ready                         worker   98m    v1.23.0
-mycluster-rhel7-2           Ready                         worker   98m    v1.23.0
-mycluster-rhel7-3           Ready                         worker   98m    v1.23.0
+mycluster-rhel8-0           NotReady,SchedulingDisabled   worker   98m    v1.23.0
+mycluster-rhel8-1           Ready                         worker   98m    v1.23.0
+mycluster-rhel8-2           Ready                         worker   98m    v1.23.0
+mycluster-rhel8-3           Ready                         worker   98m    v1.23.0
 ----
 +
 Note which machine has the `NotReady,SchedulingDisabled` status.
```

Comment:
I believe the workaround for only upgrade `NotReady,SchedulingDisabled` status RHEL worker one by one is not required for 4.9 to 4.10 upgrade, let me have a try this afternoon to confirm.

Reply:
yes, just verified upgrading two RHEL workers using the playbook from 4.9 to 4.10, they could be set in ansible inventory at the same time.

Reply:
so the section of "3. Update a RHEL worker machine:" should be like:

### 3. Update the RHEL worker machines:

#### a. Review your Ansible inventory file at /<path>/inventory/hosts and update its contents so that the RHEL-8 machines listed in the [workers] section, as shown in the following example:

```
[all:vars]
ansible_user=root
#ansible_become=True

openshift_kubeconfig_path="~/.kube/config"

[workers]
mycluster-rhel8-0.example.com
mycluster-rhel8-1.example.com
mycluster-rhel8-2.example.com
mycluster-rhel8-3.example.com
```

#### b. Change to the openshift-ansible directory:
xxx

#### c. Run the upgrade playbook:
xxx


### ~~4. Follow the process in the previous step to update each RHEL worker machine in your cluster.~~
This is not required since we have upgraded all the RHEL workers in step 3.


### 5. After you update all of the workers, confirm that all of your cluster nodes have updated to the new version:

```
# oc get node
```

Example output

```
NAME                                      STATUS                        ROLES    AGE    VERSION
mycluster-control-plane-0   Ready                         master   145m   v1.23.0
mycluster-control-plane-1   Ready                         master   145m   v1.23.0
mycluster-control-plane-2   Ready                         master   145m   v1.23.0
mycluster-rhel8-0.example.com   Ready                         master   145m   v1.23.0
mycluster-rhel8-0.example.com   Ready                         master   145m   v1.23.0
mycluster-rhel8-0.example.com.  Ready                         master   145m   v1.23.0
mycluster-rhel8-0.example.com   Ready                         master   145m   v1.23.0
```